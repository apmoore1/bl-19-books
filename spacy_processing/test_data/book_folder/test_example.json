[[1, "This is some text to test if the system works correctly."], [2, "Here is an extra test London."]]