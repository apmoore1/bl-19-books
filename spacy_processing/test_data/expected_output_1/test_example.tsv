token	pos	ner	page
<quality value="1.0" />
<token count="19" />
<book identifier="test" />
This	DT		1
is	VBZ		1
some	DT		1
text	NN		1
to	TO		1
test	VB		1
if	IN		1
the	DT		1
system	NN		1
works	VBZ		1
correctly	RB		1
.	.		1
Here	RB		2
is	VBZ		2
an	DT		2
extra	JJ		2
test	NN		2
London	NNP	GPE	2
.	.		2
