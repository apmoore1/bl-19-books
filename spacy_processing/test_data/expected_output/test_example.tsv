token	lemma	pos	ner	page
<quality value="1.0" />
<token count="19" />
<book identifier="test" />
This	this	DT		1
is	be	VBZ		1
some	some	DT		1
text	text	NN		1
to	to	TO		1
test	test	VB		1
if	if	IN		1
the	the	DT		1
system	system	NN		1
works	work	VBZ		1
correctly	correctly	RB		1
.	.	.		1
Here	here	RB		2
is	be	VBZ		2
an	an	DT		2
extra	extra	JJ		2
test	test	NN		2
London	London	NNP	GPE	2
.	.	.		2
